WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayInformationViewModel.swift

```diff
@@ -117,3 +133,16 @@ final class SetUpTapToPayInformationViewModel: PaymentSettingsFlowPresentedViewM
         }
     }
 }
+
+private extension [CardReader] {
+    func includesBluetoothReader() -> Bool {
+        return self.first(where: { reader in
+            switch reader.readerType {
+            case .appleBuiltIn:
+                return false
+            default:
+                return true
+            }
+        }) != nil
+    }
+}
```

Comment:
Nice! I didn't know we could write an extension like that, specifically for an array of a certain type 💯